documentation/CAMARA-API-Design-Guide.md

```diff
@@ -923,11 +923,31 @@ The following points can serve as a checklist to design the security mechanism o
    - With HTTP/2 or HTTP/3, performance improvements can be achieved due to better optimization and multiplexing of requests.
    - If HTTP 1.1 is used, usage of _TCP persistent connections_ is RECOMMENDED to achieve comparable performance.
 
-2. **Authentication and authorization must be considered**
+2. **Authentication and authorization**
 
-   CAMARA uses the authentication and authorization protocols and flows as described in the [Camara Security and Interoperability Profile](https://github.com/camaraproject/IdentityAndConsentManagement/blob/r3.3/documentation/CAMARA-Security-Interoperability.md).\
+   CAMARA follows the OAuth2 paradigm that separates authentication and authorization from API access.
 
-3. **Input parameter validation**
+   CAMARA uses the authentication and authorization protocols and flows as described in the released version of the [Camara Security and Interoperability Profile](https://github.com/camaraproject/IdentityAndConsentManagement/).
+   
+   All authentication and authorization decisions are bundled in an Authorization Server, which ensures that CAMARA access tokens are only then issued
+   to the API consumer,
+   - if the API consumer is registered through the onboarding process
+   - if the API consumer is using the authorization grant/flow agreed upon during the onboarding process
+   - if and after the User or API consumer sucessfully authenticated to the Authorization Server
+   - if the requested Purpose and scopes were agreed upon during the onboarding process
+  
+   This separation of concerns simplifies the implementation of the API endpoint that, after validating the access token, can now focus on its API implementation.
+
+   The access token is created by the API provider's Authorization Server to be consumed by the API provider's API endpoint.
+
+   As described in [Appendix A (Normative): `info.description` template for when User identification can be from either an access token or explicit identifier](#appendix-a-normative-infodescription-template-for-when-user-identification-can-be-from-either-an-access-token-or-explicit-identifier) a three-legged access token is associated with Personal Identifiable Information (PII).
+
+   If the access token is self-contained then the three-legged access token contains PII and the confidentiality of that PII must be protected.
+   The confidentiallity of the PII in the self-contained access token can be achieved by the API provider's authorization server encrypting 
+   the access token to the API Provider's API endpoint.
+   It is recommended that self-contained access tokens follow RFC 9068 JSON Web Token (JWT) Profile for OAuth 2.0 Access Tokens.

```

Comment:
This RFC is not referenced in Identity and Consent Management WG, so it is weird to refer it in Commonalities whilst not reflected in any ICM documentation.

I think it should be first aligned in ICM before making any indication in Commonalities indication

Reply:
I removed that recommandation